### DISCUSSION TITLE
When will Theia upgrade to latest vscode version

### DISCUSSION THREAD

**Discussion by ghost** [eclipse-theia/theia#11781] [General] [answered]
Hi

We have a vscode extension which we are trying to use in theia.
Our vscode extension uses APIs from vscode 1.67.2 
I guess theia supports vscode api version 1.53.2.
So,many of our features don't work.
Do you know when will theia move to later version of vscode api like 1.67.2

**Comment by msujew** [comment, marked as answer]
Hey @badrirav,

we have some trackers for our extension API milestones, such as https://github.com/eclipse-theia/theia/issues/9863, https://github.com/eclipse-theia/theia/issues/11520 and https://github.com/eclipse-theia/theia/issues/11779. The support for `1.55.2` is almost finished, and might be included in this month's releases.

Note that you can always contribute newer parts of the extension API to Theia yourself if you like. Even if the officially supported version is `1.53.2`, this is only an indicator for *full support* of the API. We have partial support of many newer versions. So if you are only missing some minor parts of the API, you should be able to still run your vscode extension, as long as the parts that it needs are actually implemented.
